src/test/java/core/basesyntax/handlers/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.storage.Storage;
+import core.basesyntax.transaction.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationHandlerTest {
+    private static final String APPLE = "apple";
+    private static final int INITIAL_FRUIT_QUANTITY = 100;
+    private static final int PURCHASE_QUANTITY = 50;
+    private static final String BANANA = "banana";
+    private static final int INVALID_FRUIT_QUANTITY = -10;
+    private static FruitTransaction validFruitTransaction;
+    private static FruitTransaction invalidFruitTransaction;
+    private static PurchaseOperationHandler purchaseOperationHandler;
+
+    @BeforeAll
+    static void beforeAll() {
+        purchaseOperationHandler = new PurchaseOperationHandler();
+
+        validFruitTransaction = new FruitTransaction();
+        validFruitTransaction.setFruit(APPLE);
+        validFruitTransaction.setOperation(FruitTransaction.Operation.PURCHASE);
+        validFruitTransaction.setQuantity(PURCHASE_QUANTITY);
+
+        invalidFruitTransaction = new FruitTransaction();
+        invalidFruitTransaction.setFruit(BANANA);
+        invalidFruitTransaction.setOperation(FruitTransaction.Operation.PURCHASE);
+        invalidFruitTransaction.setQuantity(INVALID_FRUIT_QUANTITY);
+
+        Storage.getFruitBalance().put(APPLE, INITIAL_FRUIT_QUANTITY);
+        Storage.getFruitBalance().put(BANANA, INITIAL_FRUIT_QUANTITY);
+    }
+
+    @AfterEach
+    void afterEach() {
+        Storage.getFruitBalance().clear();
+    }

```

Comment:
Let's have a look at that code.
You store apple and bana before all tests. Then you will execute first test and after it you will clear storage.
So, for the second test, you storage will be clear.
Let's make your storage same for all tests or add some consistency here.